backend/core/src/translations/entities/translation.entity.ts

```diff
@@ -0,0 +1,36 @@
+import { Column, Entity, Index } from "typeorm"
+import { AbstractEntity } from "../../shared/entities/abstract.entity"
+import { Expose } from "class-transformer"
+import { IsEnum, IsString, MaxLength } from "class-validator"
+import { ValidationsGroupsEnum } from "../../shared/validations-groups.enum"
+import { ApiProperty } from "@nestjs/swagger"
+import { CountyCode } from "../../shared/types/county-code"
+import { Language } from "../../shared/types/language-enum"
+
+@Entity()
+@Index(["countyCode", "language", "key"], { unique: true })
```

Comment:
I don't think we need `key` here. To get keys one by one, would require many requests to db. If we use just countyCode and language as keys and have a json field holding all the translations for county I think that would be enough. One call to get all of the needed translations.

Reply:
Applied comments, thanks